public_html/wp-content/plugins/camptix/addons/require-login.php

```diff
@@ -71,7 +71,11 @@ public function block_unauthenticated_actions() {
 			return;
 		}
 
-		if ( ! is_user_logged_in() ) {
+		// Temporary: We don't want to block users from editing tickets.
+		// See: https://github.com/WordPress/wordcamp.org/issues/1393.
+		$user_is_editing_ticket = in_array( $_REQUEST['tix_action'], array( 'access_tickets', 'edit_attendee' ) );
```

Comment:
Ideally we should check that the ticket doesn't have an assigned username already, since the workaround should only cover those tickets that were purchased before this was possible.

Reply:
I agree, but I'm not overly worried about that myself.  If someone has the ticket access (or edit) token even if they're logged into the wrong account I don't see that as being a major concern.